PIL/ImageOps.py

```diff
@@ -178,6 +178,27 @@ def crop(image, border=0):
         )
 
 
+def scale(self, factor, resample=Image.NEAREST):
+    """
+    Returns a rescaled image by a specific factor given in parameter.
+    A factor greater than 1 expands the image, between 0 and 1 contracts the
+    image.
+
+    :param factor: The expansion factor, as a float.
+    :param resample: An optional resampling filter. Same values possible as
+       in the PIL.Image.resize function.
+    :returns: An :py:class:`~PIL.Image.Image` object.
+    """
+    if factor == 1:
+        return self._new(self.im)
```

Comment:
I agree with you. I think it's faster to return the value here rather make the calculations of the destination size, pass the relay to the resize function, and then return the value because factor = 1.
Not tested in practice however.